src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -1,36 +1,28 @@
-import React, { ChangeEvent, FocusEvent, ReactNode, useContext } from 'react'
+import React, { ReactNode, useContext } from 'react'
 import classnames from 'classnames'
 import ContextProvider from 'frontend/state/ContextProvider'
 import './index.css'
 
-interface TextInputFieldProps {
+interface TextInputFieldProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
   htmlId: string
-  value: string
-  onChange: (event: ChangeEvent<HTMLInputElement>) => void
   inputIcon?: ReactNode
   afterInput?: ReactNode
   label?: string
   placeholder?: string
-  disabled?: boolean
   extraClass?: string
   warning?: ReactNode
-  onBlur?: (event: FocusEvent<HTMLInputElement>) => void
-  maxLength?: number
 }
```

Comment:
Fixed up the types here a bit, that way it's actually an input component with all html props supported